runtime/oti/vm_api.h

```diff
@@ -356,12 +356,13 @@ internalCreateArrayClass(J9VMThread* vmThread, J9ROMArrayClass* romClass, J9Clas
  * @param className String object representing name of the class to load
  * @param classLoader J9ClassLoader to use
  * @param options load options such as J9_FINDCLASS_FLAG_EXISTING_ONLY
+ * @param expectedReturnCode the expected return code to the caller
```

Comment:
The parameter name is confusing. Please rename it and change the documentation to indicate that it is the set of allowed bits.

Reply:
Updated the documentation against the suggestion above.